simplehound/core.py

```diff
@@ -45,6 +51,20 @@ def encode_image(image: bytes) -> str:
     """base64 encode an image."""
     return base64.b64encode(image).decode("ascii")
 
+def get_licensePlates(detections: Dict) -> List[Dict]:
+    """
+    Get the list of the faces.
+    """
+    licensePlates = []
+    for obj in detections["objects"]:
+        if not obj["objectType"] == "licenseplate":
+            continue
+        plate = {}
+        plate["name"] = obj["licenseplateAnnotation"]["attributes"]["system"]["string"]["name"]
+        plate["confidence"] = obj["licenseplateAnnotation"]["attributes"]["system"]["string"]["confidence"]
+#        plate["boundingBox"] = obj["licenseplateAnnotation"]["bounding"]["system"]["vertices"]
```

Comment:
@robmarkcole need guidance on converting vertices so that I can get bounding boxes when the images get saved.